test/helpers/local_time_helper_test.rb

```diff
@@ -17,6 +17,11 @@ class LocalTimeHelperTest < TestCase
   include ActionView::Helpers::DateHelper, ActionView::Helpers::TagHelper
   include LocalTimeHelper
 
+  def normalize(html)
+    n = Nokogiri::HTML(html).xpath('//time').first
```

Comment:
I'm not a big fan of using `n` or `x` etc. in variable naming. It makes stuff kind of unclear, especially in manipulation like you are doing later on ( where you are using `n` again ). I would recommend using something like `document` here and then getting a little bit more descriptive in `#map`'s vars.